types/src/account_config/resources/core_account.rs

```diff
@@ -97,3 +97,15 @@ impl MoveStructType for AccountResource {
 }
 
 impl MoveResource for AccountResource {}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_account_resource_has_special_address() {
```

Comment:
oh I see :-) maybe we can refer to this assumption / test where we do the special check as well.